### ISSUE TITLE
Upgradable contract

### ISSUE THREAD

**Issue by alexwasner** [mouse-dev-1/Akutar#8]
Y'all need to make this contract upgradable. It could've potentially saved the day with the original contract. https://docs.openzeppelin.com/contracts/3.x/upgradeable

**Comment by sonya75** [comment]
One could easily argue that the reason for not making a contract upgradable is to make it trustworthy. If it's upgradeable, that gives the owner full access to change anything he wants at any point. A contract like this is a very small piece of code. It's quite possible to carefully read it and simulate everything to make sure there isn't any flaw in it.

**Comment by 0xildefonso** [comment]
agree, it would be great if the original DA contract is upgradable. 
tho doesn't make much sense to make this contract upgradable as it's relatively simple
